src/Network/Send/kRO/Sakexe_2004_07_05a.pm

```diff
@@ -18,7 +18,7 @@
 package Network::Send::kRO::Sakexe_2004_07_05a;
 
 use strict;
-use base qw(Network::Send::kRO::Sakexe_0);
+use base qw(Network::Send::ServerType0);
```

Comment:
I don't think you should be doing this

Reply:
@lututui  I'm lazy to update the sakexe_0 part. It's missing too many packets.
Update just servertybe0 it would be better, right?

Reply:
You don't know what this implies. All RagexeRE and Sakexe would become ST0 children.

Reply:
I think that's good too.

Reply:
Does recv use qw sT0 and send using Sakexe_0? @lututui